Carbon/Functions/Install-WindowsFeature.ps1

```diff
@@ -1,6 +1,6 @@
 
 # This function should only be available if the Windows PowerShell v3.0 Server Manager cmdlets aren't already installed.
-if( -not (Get-Command -Name 'Get-CWindowsFeature*' | Where-Object { $_.ModuleName -ne 'Carbon' }) )
+if( -not (Get-Command -Name 'Get-WindowsFeature*' | Where-Object { $_.ModuleName -ne 'Carbon' }) )

```

Comment:
Did you mean to check for `Get-WindowsFeature` or `Install-WindowsFeature`?

Reply:
By design.